NitroxServer/Communication/LiteNetLib/LiteNetLibServer.cs

```diff
@@ -122,13 +120,12 @@ private void PeerDisconnected(NetPeer peer, DisconnectInfo disconnectInfo)
 
     private void NetworkDataReceived(NetPeer peer, NetDataReader reader, byte channel, DeliveryMethod deliveryMethod)
     {
-        netPacketProcessor.ReadAllPackets(reader, peer);
-    }
+        int packetDataLength = reader.GetInt();
+        byte[] packetData = new byte[packetDataLength];
```

Comment:
We might wanna check if packetDataLength hasn't a weird value before allocating memory based on this int